probe/kubernetes/controls.go

```diff
@@ -52,6 +53,16 @@ func (r *Reporter) deletePod(req xfer.Request, namespaceID, podID string, _ []st
 	}
 }
 
+// DeletePersistentVolumeClaim is the control to delete kubernetes persistentvolumeclaim
+func (r *Reporter) DeletePersistentVolumeClaim(req xfer.Request, namespaceID, persistentvolumeclaimID string) xfer.Response {
```

Comment:
This function name should start with small letters as Weave is using the same way.
This is also required because we have two functions with the same name.